scripts/histogramming.py

```diff
@@ -0,0 +1,45 @@
+import pickle
+import histogramming_config as hf
+import Histogramming.storage_functions as sf
+
+def run():
+
+    print('test')
+    
+    histogramming = sf.HistoMaker()
+
+    client = histogramming.client_start(**hf.client_params)
+
+    print(f'Client Dashboard: {client.dashboard_link}')
+
+    x = histogramming.create_file_list(**hf.file_list)
+
+    histogramming.compute_histograms(**hf.computation_params)
+
+    histograms_list, histograms_dict = histogramming.combine_histograms()
+
+    print('histogram of rljet_pt_comb')
+    print(histograms_dict['rljet_pt_comb'])
+
+    for key in histograms_dict:
+
+        with open(f"{hf.out_dir}/{key}_file.pkl", "wb") as f:
```

Comment:
The path to inputs should be in the config. Why is there pkl file reading here?

Reply:
this is just as an example for a user it can be changed in config